third_party/BUILD

```diff
@@ -14,35 +14,45 @@
 
 package(default_visibility = ["//visibility:public"])
 
+# DO NOT SUBMIT: audit all of the tags here
+
 java_library(
     name = "jsr250_annotations",
+    tags = ["maven_coordinates=javax.annotation:jsr250-api:1.0"],
     exports = ["@javax_annotation_jsr250_api//jar"],
 )
```

Comment:
I also think, that `maven_jar` (both the native one and the Skylark based) and `java_import_external` automagically adding `maven_coordinates=foo:bar:baz? That way you wouldn't need to abuse tags and more importantly, wouldn't need to duplicate maven coordinates and versions. Let's face it, this would be a maintenance mess. I already see, that both coordinates and version would be changed in once place but would be forgotten to be changed in another, so that this would end up in a mismatch between real dependencies and the ones exposed in generated pom.xml files.

Reply:
Yup, that's part of my proposal in https://github.com/bazelbuild/bazel/issues/1372. I don't think these should be regularly hardcoded except for exceptional cases.